datacite-findable-doi-handler/src/test/resources/doi_publication_event_v2.json

```diff
@@ -0,0 +1,18 @@
+{
+  "detail": {
+    "requestPayload": {},
+    "responseContext": {},
+    "responsePayload": {
+      "type": "publication.doiupdate.request",
+      "item": {
+        "metadata": "<xml>",
```

Comment:
I was working a new event type where the publication object was changed to more explicit fields. Logic for populating these fields must be moved into the code producing the event which is publication service. This would remove all dependencies on datamodel in this service.